config/config.h.in

```diff
@@ -57,6 +57,8 @@
 #cmakedefine USE_FNAME_CASE
 #define USEMAN_S 1
 
+#cmakedefine USE_CRNL
```

Comment:
Vim defines this in `os_dos.h`, so let's stick with `win_defs.h`, it saves indirection.